docs/source-app/workflows/ssh/index.rst

```diff
@@ -0,0 +1,124 @@
+
+#################################
+Debug cloud apps via SSH
+#################################
+
+**Audience:** Users that want to debug their cloud apps on their own machine.
+
+SSH allows you to access all your Lightning components in real time.
+This can be useful if you execute your app in a different environment than you develop it.
+This is often the case with applications that rely on GPUs.
+
+You can inspect the entire file system and monitor the runtime environment.
+
+----
+
+**********************************************************
+Add SSH key to lightning
+**********************************************************
+
+Before you can SSH to cloud machines, you will need to generate a new private SSH key, add it to the SSH agent, and add the public SSH key to your account on Lightning.
+
+
+Step 1: Create an SSH key
+==========================
+
+Open a terminal and run the following command (replace email with the address you used in your lightning.ai account):
+
+.. code:: bash
+
+   # make the ssh key (if you don't have one)
+   $ ssh-keygen -t ed25519 -C "your_email@example.com"
+
+This creates a new SSH key, using the provided email as a label.
+
+At the prompt, type a secure passphrase.
+
+.. code:: bash
+
+   > Enter passphrase (empty for no passphrase): [Type a passphrase]
+   > Enter same passphrase again: [Type passphrase again]
+
+
+Step 2: add the key to the ssh-agent
```

Comment:
I find it strange that this step is in this section. This step is not a one-time setup and needs to be repeated every time you re-login to your system. I would expect it to be mentioned in the section where we ssh into the machine.

Reply:
I was skeptical about even adding this in the first place, as it is setup dependent.
Ie, with a hardware key like a yubikey, you don't need to do anything.

I'll move it around, as that is an uncommon setup I think.